script.js

```diff
@@ -44,6 +44,21 @@ idApp.eventListeners = function() {
       }
    })
 
+   // Event Listener #3A for when any radio input is selected to display a div with some more info
+   $main.on('click', '.optionContainer', function () {
+      $('.hiddenContents').removeClass('displayContents');
+      $('.fa-plus').removeClass('active')
+      $(this).find('.hiddenContents').toggleClass('displayContents');
+      $(this).parent('.windowBLabel').find('.fa-plus').addClass('active')
+   })
+
+   // Event Listener #3B to hide the extra info by clicking the X button
```

Comment:
Love the 'X' idea for getting users to close/hide the extra info. It's intuitive, and great that we don't have to add extra instructions at the top